src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -137,6 +139,37 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
         this.fetchCurrentUser(currentUser);
     }
 
+    componentDidUpdate({ annotatorState: prevAnnotatorState }: Props): void {
+        const { error: prevError } = prevAnnotatorState;
+        const {
+            annotatorState: { error },
+        } = this.props;
+
+        if (error && prevError !== error) {
+            switch (error.code) {
```

Comment:
Do we expect further codes to be handled here? It seems like this could be simplified.